app/models/client_opportunity_match.rb

```diff
@@ -391,6 +391,13 @@ def activate!
     end
   end
 
+  def matched!
+    self.class.trasaction do
+      add_default_contacts!
+      self.send(match_route.initial_decision).initialize_decision!
```

Comment:
This is going to "start" the match, putting the first decision into the `:pending` state.  I don't think we want that to happen until the match is activated.